packages/calling-stateful-client/src/CallDeclarativeCommon.ts

```diff
@@ -122,6 +123,7 @@ class ProxyTeamsCaptionsFeature implements ProxyHandler<TeamsCaptionsCallFeature
           async (...args: Parameters<TeamsCaptionsCallFeature['stopCaptions']>) => {
             const ret = await target.stopCaptions(...args);
             this._context.setIsCaptionActive(this._call.id, false);
+            this._context.setStartCaptionsInProgress(this._call.id, false);
```

Comment:
Is it possible that caption will be always in progress and never return back to normal? How do we handle scenarios like this?

Reply:
That means start captions failed and it will stay on the in progress screen, will see error in console too similar to what happens when start call fail